<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-E7CUST6" failures="0" tests="11" name="TestPractice.TestNGKeywordProgram" time="2.049" errors="1" timestamp="2025-01-24T11:09:30 IST" skipped="2">
  <testcase classname="TestPractice.TestNGKeywordProgram" name="selectItem" time="0.002"/>
  <testcase classname="TestPractice.TestNGKeywordProgram" name="selectItem" time="0.002"/>
  <testcase classname="TestPractice.TestNGKeywordProgram" name="selectItem" time="0.006"/>
  <testcase classname="TestPractice.TestNGKeywordProgram" name="selectItem" time="0.004"/>
  <testcase classname="TestPractice.TestNGKeywordProgram" name="checkRate" time="0.004"/>
  <testcase classname="TestPractice.TestNGKeywordProgram" name="addToCart" time="2.021">
    <error message="Method TestPractice.TestNGKeywordProgram.addToCart() didn&#039;t finish within the time-out 2000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method TestPractice.TestNGKeywordProgram.addToCart() didn't finish within the time-out 2000
at java.base@17.0.8.1/java.lang.Thread.sleep(Native Method)
at app//TestPractice.TestNGKeywordProgram.addToCart(TestNGKeywordProgram.java:61)
at java.base@17.0.8.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@17.0.8.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base@17.0.8.1/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@17.0.8.1/java.lang.reflect.Method.invoke(Method.java:568)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@17.0.8.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.8.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.8.1/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- addToCart -->
  <testcase classname="TestPractice.TestNGKeywordProgram" name="checkoutItem" time="0.001">
    <skipped/>
  </testcase> <!-- checkoutItem -->
  <testcase classname="TestPractice.TestNGKeywordProgram" name="selectItem" time="0.001"/>
  <testcase classname="TestPractice.TestNGKeywordProgram" name="searchTest" time="0.005"/>
  <testcase classname="TestPractice.TestNGKeywordProgram" name="advancedSearchTest" time="0.003"/>
  <testcase classname="TestPractice.TestNGKeywordProgram" name="verifyItemCount">
    <skipped/>
  </testcase> <!-- verifyItemCount -->
</testsuite> <!-- TestPractice.TestNGKeywordProgram -->
